stablehlo/dialect/TypeInference.cpp

```diff
@@ -2649,9 +2649,24 @@ LogicalResult inferSelectOp(
 
   // The output shape should be derived from the most specific parts of the
```

Comment:
We can update the comment here with something like 
`The output shape ..... when the bounds are missing`
(same below)

Also `see documentation for details`:  I think I am missing which documentation? same below

Reply:
Sure. Re: documentation, I clarified to refer to `inferXXX`'s documentation.